### PR TITLE
Turn to angle fix

### PR DESCRIPTION
**Pull request by gaudetja**
Gripper opens prematurely in autonomous, should score on scale otherwise during auto

### CODE REVIEW

**Comment by jkraljic** [comment]
Thanks for the update.

File: PowerUp/src/Commands/AutonomousDriveWithEncoders.cpp [hunk 1]
```diff
@@ -5,42 +5,53 @@ AutonomousDriveWithEncoders::AutonomousDriveWithEncoders(double metersToDrive, d
...
-	Robot::driveTrain->SetCoast();
-	Robot::driveTrain->ArcadeDrive(0, 0, true);
+	RobotMap::ahrs->Reset();
 	RobotMap::ahrs->ResetDisplacement();
-//	Wait(2);
+
+	Robot::driveTrain->ArcadeDrive(0, 0, false);
+
+	// maybe wait to give it time to brake before coasting again
+	Wait(50);
```
Comment (snutt) [inline PowerUp/src/Commands/AutonomousDriveWithEncoders.cpp]: What does the `Wait(50)` do? I don't think we want to pause the main thread

Comment (gaudetja) [reply PowerUp/src/Commands/AutonomousDriveWithEncoders.cpp]: I think I made a bad merge and this is why I could not drive with encoders yesterday!

**Comment by rpbritton** [comment]
👍

**Pull request closed without merging**